apps/betterangels-backend/clients/tests/utils.py

```diff
@@ -71,7 +70,6 @@ def setUp(self) -> None:
             }
             hmisProfiles {
                 id
-                hmisId
```

Comment:
I don’t think we want to remove this field—this is removing hmisId from the HmisProfile, which I believe is part of the new method. @mikefeldberg, can you confirm?